peachjam/templates/peachjam/_document_table.html

```diff
@@ -20,7 +20,15 @@
           {% else %}
             <td >{{ document.author|default_if_none:'' }}</td>
           {% endif %}
-          <td style="white-space: nowrap;">{{ document.date }}</td>
+            <td style="white-space: nowrap;">{{ document.date }}</td>
+
+            <td style="white-space: nowrap;">
+              {% if document.work.languages|length > 1 %}
+                <span class="d-inline-block" tabindex="0" data-toggle="tooltip" title="Multi-language">
```

Comment:
The string must be translatable - "a multiple languages available" is more descriptive, too.